src/sentry/templates/sentry/emails/data-export-finished.txt

```diff
@@ -0,0 +1,5 @@
+We've finished assembling your download, check it out by clicking the link below:
+
+{{url|safe}}
```

Comment:
I'm not really sure what `|safe` means, but I noticed it on a few other text templates, if someone knows why/why not it should be included please let me know 👍

Reply:
@leeandher I believe it refers to [preventing html escaping on your url](https://docs.djangoproject.com/en/3.0/ref/templates/builtins/#safe). Ex: `&` becomes `&amp;`. I think it's better to keep it there.

Reply:
Ahhh that makes a lot of sense, thanks!